mitodl/bootcamp-ecommerce
# Fixes faulty logic in Profile.is_complete

#### What are the relevant tickets?
Fixes #875

#### What's this PR do?
Fixes profile complete logic so it doesn't reject `0` values

#### How should this be manually tested?
Run `is_complete` for some `Profile` object with `company_size` set to 0. As long as everything else is filled out, it should return `True`
